db/dcrsqlite/apisource.go

```diff
@@ -1117,6 +1178,26 @@ func (db *wiredDB) GetExplorerTx(txid string) *explorer.TxInfo {
 	return tx
 }
 
+func (db *wiredDB) GetAddressUTXO(address string) []*apitypes.AddressTxnOutput {
```

Comment:
why implement this for `wiredDB` too?  Do we intend for this to be used in lite mode as well as full with PG?  If so, it's time to write an function that repeatedly calls `SearchRawTransactionsVerbose` until it gets all the transactions, although we've found that is very demanding on dcrd.